### PR TITLE
Part 1 of addressing # 13

### PR DESCRIPTION
**Pull request by arunkoshy**
- kicks off work on https://github.com/opencomputeproject/OSF-OSFCI/issues/13
- provides a sample file update for recent work done with IP and email filters

Signed-off-by: Arun Koshy <arunkoshy@gmail.com>

### CODE REVIEW

**Comment by arunkoshy** [comment]
@vejmarie this should be good to merge as changes are already running on test infra and the sample has been okayed by @susmithp

File: config/prohibited.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+BANNED_DOMAINS: nsogroup.com,irangov.ir
```
Comment (vejmarie) [inline config/prohibited.yaml:2]: Can we use non-existing domain as example ? Something like <prohibiteddomain.com, notagoodfriend.com>

Comment (arunkoshy) [reply config/prohibited.yaml:2]: @vejmarie sure anything you want ;-)

**Review by susmithp — COMMENTED** [review]
Looks good to me.

**Pull request merged**